### PR TITLE
DEOPS-134 Add docs describing schema source directories and files.

### PR DESCRIPTION
**Pull request by psarando**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,103 @@ Discovery Environment Database Schema
...
+
+The [extensions](src/main/extensions) directory contains files defining PostgreSQL extensions required for this database.
+These should only need to be defined once, when the database is initially setup, so these files should be loaded first.
+
+### Tables
+
+The [tables](src/main/tables) directory contains files defining all tables in this database.
+
+Each file should be named the same as the table it defines.
+These files used to also define constraints for each table, which is why they are prefixed by a number
```
Comment (slr71) [inline README.md]: It might be useful to replace "which is why they are prefixed" with "which is why existing files are prefixed"

**Review by slr71 — APPROVED** [review]
This sounds good to me. Thanks for writing this; it'll help to make it easier to create database conversions in the future. I had one minor wording suggestion, but that was it. Feel free to :shipit: whenever you're ready.

**Review by johnworth — APPROVED** [review]
LGTM :shipit:

**Comment by psarando** [comment]
Thanks for the reviews 👍

**Pull request merged**